src/main/java/wolox/training/models/Book.java

```diff
@@ -10,50 +10,51 @@
 import javax.persistence.Id;
 import javax.persistence.ManyToMany;
 import javax.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
 import wolox.training.models.DTO.BookDTO;
 import wolox.training.utils.Constants;
 
 @Entity
+@NoArgsConstructor
 public class Book {
 
     @Id
     @GeneratedValue(strategy = GenerationType.SEQUENCE)
-    private long id;
+    @Getter private long id;
```

Comment:
Checkout the `@Data` annotation, it includes the setters, getters, .toString(), .equals() and .hashCode() methods :) 
You can use `@Setter(AccessLevel.NONE)` for the id specifically, to avoid including a setter for that one attribute.